src/AutoRest.CSharp/Mgmt/AutoRest/MgmtConfiguration.cs

```diff
@@ -265,7 +265,7 @@ internal static MgmtConfiguration GetConfiguration(IPluginCommunication autoRest
                 operationPositions: autoRest.GetValue<JsonElement?>("operation-positions").GetAwaiter().GetResult(),
                 requestPathToSingletonResource: autoRest.GetValue<JsonElement?>("request-path-to-singleton-resource").GetAwaiter().GetResult(),
                 overrideOperationName: autoRest.GetValue<JsonElement?>("override-operation-name").GetAwaiter().GetResult(),
-                renameRules: autoRest.GetValue<JsonElement?>("rename-rules").GetAwaiter().GetResult(),
+                renameRules: GetAcronymMappingConfig(autoRest),
```

Comment:
Could we rename the name of the argument here as well?
```suggestion
                acronymMapping: GetAcronymMappingConfig(autoRest),
```